lib/dependencies/LocalModulesHelpers.js

```diff
@@ -40,4 +20,29 @@ function lookup(parent, mod) {
 	}
 
 	return path.join("/");
+};
+
+class LocalModulesHelpers {
+
+	static addLocalModule(state, name) {
+		if(!state.localModules) state.localModules = [];
+		var m = new LocalModule(state.module, name, state.localModules.length);
+		state.localModules.push(m);
+		return m;
+	}
+
+	static getLocalModule(state, name, namedModule) {
+		if(!state.localModules) return null;
+		if(namedModule) {
+			// resolve dependency name relative to the defining named module
+			name = lookup(namedModule, name);
+		}
+		for(var i = 0; i < state.localModules.length; i++) {
+			if(state.localModules[i].name === name)
+				return state.localModules[i];
+		}
+		return null;
+	}
 }
+
+module.exports = LocalModulesHelpers;
```

Comment:
no, It doesn't. If you try that it will breaks:

```javascript
class Triple {
  static triple(n) {
    if (n === undefined) {
      n = 1;
    }
    return n * 3;
  }
}

class BiggerTriple extends Triple {
  static triple(n) {
    return super.triple(n) * super.triple(n);
  }
}

console.log(Triple.triple());        // 3
console.log(Triple.triple(6));       // 18

var tp = new Triple();

console.log(BiggerTriple.triple(3));
// 81 (not affected by parent's instantiation)

console.log(tp.triple());
// 'tp.triple is not a function'.
```

More details about it on MDN static documentation (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/static).

If you look the code, we didn't change the current behavior that we already have, so if we don't need to expose as functions because they already been used as classes and static methods. The same is applied to other files as well.

Based on the scenario that we have now this change could be good since we are using as a class, conceptually talking, we should make it explicit.